clang-tools-extra/clangd/unittests/CompileCommandsTests.cpp

```diff
@@ -526,6 +526,16 @@ TEST(CommandMangler, RespectsOriginalSysroot) {
                 Not(HasSubstr(testPath("fake/sysroot"))));
   }
 }
+
+TEST(CommandMangler, StdLatestFlag) {
```

Comment:
As mentioned in the original PR, I'm not sure if this is the best test case for this change, because the fact that `transferCompileCommand()` drops `/std:c++latest` is itself a bug which will be fixed in https://github.com/llvm/llvm-project/pull/160030.

But I'm not sure how to write a more direct test for "`transferCompileCommand()` was not called", and the scenario being tested here is a valid one from a user expectation point of view.